### PR TITLE
Remove ipywidgets pin

### PR DESCRIPTION
**Pull request by hadim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hadim** [comment]
nglview >=3.0.4 supports latest ipyqidgets in both notebook and jlab: https://github.com/nglviewer/nglview/releases

**Comment by hadim** [comment]
@conda-forge-admin, please rerender

**Comment by hadim** [comment]
@conda-forge/openff-toolkit ready for review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
     sha256: a1a4647ecaae364a8c8321c4c792d68f0336588c435156d24ac5153eb88b9032
 
 build:
-  number: 0
+  number: 10
```
Comment (mattwthompson) [inline recipe/meta.yaml:14]:
```suggestion
  number: 1
```

**Review by mattwthompson — CHANGES_REQUESTED** [review]

**Comment by mattwthompson** [comment]
@Yoshanuikabundi found an issue with some combination of NGLview, JupyterLab, and Colab, but I think that's a compatibility issue with the latter two that we're not dealing with here, so this change should be fine, right Josh?

**Comment by Yoshanuikabundi** [comment]
I thought NGLView didn't work with JupyterLab 4.x, but it looks like it does work with lab 4 locally if I specify IPyWidgets 8.x. I thought I had tested it with an environment that was just Lab and NGLview but I guess I didn't. Colab doesn't seem to work in either case but now that I know this I'm making progress on that.

In other words this change may actually fix that issue!

**Comment by mattwthompson** [comment]
Let's roll with this for now, then. While we're unsure of limitations we should be permissive

**Review by mattwthompson — APPROVED** [review]

**Comment by j-wags** [comment]
Perfect - Thanks everyone!

**Pull request merged**